src/soilbiogeochem/SoilBiogeochemNitrifDenitrifMod.F90

```diff
@@ -168,10 +168,11 @@ subroutine SoilBiogeochemNitrifDenitrif(bounds, num_soilc, filter_soilc, &
     real(r8) :: mu, sigma
     real(r8) :: t
     real(r8) :: pH(bounds%begc:bounds%endc)
+    real(r8) :: D0  ! temperature dependence of gaseous diffusion coefficients
     !debug-- put these type structure for outing to hist files
     real(r8) :: co2diff_con(2)                      ! diffusion constants for CO2
-    real(r8) :: eps
-    real(r8) :: f_a
+    real(r8) :: fc_air_frac ! Air-filled fraction of soil volume at field capacity
```

Comment:
are we trying to line up comments for variable descriptions any more?

Reply:
I thought we decided that was more trouble than it's worth; see "Lower priority" section of [notes from 2023-09-07](https://docs.google.com/document/d/1Qn9c7NiGxYMN3p47Fpa0QINLwwxzRpGXG61GsuGjVM0/edit#heading=h.3vr45vczwma).

Reply:
Yeah, when I see what that actually looks like in practice I wonder if it's the right decision? 
- I thought when @ekluzek raised the question it was WRT going through and correcting existing code that's not being modified (except for the vertical alignment).  Avoiding this seems OK to prevent scope creep on our work.  
- Introducing new text that's not aligned (especially in code blocks that are otherwise tidy and easy to read) seems misguided.
Just my 2-cents, but I suggest aligning this now, merging the PR and discussing the topic at an SE meeting.  This is bleeding into a discussion about 'reflections on our process', with is outside the scope of a bug fix.

Reply:
Done! (2bb68ad)